ppx-lib/gen_js_api_ppx.ml

```diff
@@ -1509,24 +1530,28 @@ and mapper =
     let e = super.expr self e in
     let global_attrs = [] in
     match e.pexp_desc with
-    | Pexp_extension (attr, PTyp ty) when filter_attr_name "js.to" attr ->
+    | Pexp_extension (attr, PTyp ty) when filter_extension "js.to" attr ->
         with_default_loc e.pexp_loc
           (fun () -> js2ml_fun (parse_typ ~global_attrs ty))
-    | Pexp_extension (attr, PTyp ty) when filter_attr_name "js.of" attr ->
+    | Pexp_extension (attr, PTyp ty) when filter_extension "js.of" attr ->
         with_default_loc e.pexp_loc
           (fun () -> ml2js_fun (parse_typ ~global_attrs ty))
     | _ ->
         e
   in
-  {super with module_expr; structure_item; expr; typ}
+  let attribute self a =
+    ignore (filter_attr "js.dummy" a : bool);
```

Comment:
Can you say a few words about this change ?

Reply:
One of the reason I want to integrate with ppxlib is that ppxlib will complain about any unused attributes. In that context, using an arbitrary attribute for defining `functions that return functions` does not work. I've created a special dummy attribute for that case.